labs/03-deletion.md

```diff
@@ -9,31 +9,42 @@ Spin up the container with `docker run -ti alpine`
 list all the folders on the root level to see the whole distribution:
 
 ```
-# ls /
+ls /
+```
+
+Expected output:
+
+```
 bin    etc    lib    mnt    root   sbin   sys    usr
 dev    home   media  proc   run    srv    tmp    var
 ```
 
-> **Warning:** Do never try to run the following command as a super user in your own environment, as it will delete _everything_ on your computer.
+> **Warning:** Do not run the following command as a super user in your own environment, as it will delete _**everything**_ on your computer.
 
-Now, delete the whole file system with `rm -rf /`
+Now, delete the whole file system with
 
-Try to navigate around to see how much of the OS is gone
+```
+rm -rf /
+```
+
+Try to navigate around to see how much of the OS is gone: run the `ls`, `whami` and `date`.
+They should all echo back that the binary is not found.
+
+Exit out by pressing `Ctrl+d` and create a new instance of the Alpine image and look a bit around:
 
 ```
-# ls
```

Comment:
Why are we deleting this part?

Reply:
and the 5 lines below as well.

Reply:
In an attempt to not make the "shell snippets" confusing, and to be consistent with the other shell snippet formats. 

The commands to run are listed above.

Reply:
Ahh sure, did not see the listing of the commands :+1: